samples/arch/smp/pktqueue/sample.yaml

```diff
@@ -19,7 +19,7 @@ tests:
       - smp
     filter: (CONFIG_MP_MAX_NUM_CPUS > 1)
     platform_exclude:
-      - esp32
+      - esp32_devkitc_wroom
```

Comment:
Such changes should be done together with the commit that split `esp32` into two separate platforms.

Reply:
Answered that in the later comment.